R/prob-brier.R

```diff
@@ -83,6 +83,10 @@ brier_class_vec <- function(truth,
                             na_rm = TRUE,
                             case_weights = NULL,
                             ...) {
+  if (is_class_pred(truth)) {
```

Comment:
Does this metric use the hard class predictions?

Reply:
Ok, I see that a lot of other probability metrics have this. I don't think that they should/do use this column.

Reply:
Done in [tidymodels/yardstick@`0352475` (#422)](https://github.com/tidymodels/yardstick/pull/422/commits/0352475d7273b4017465db775728043ceca492d0)